LT.DigitalOffice.Kernel.nuspec

```diff
@@ -2,18 +2,17 @@
 <package >
   <metadata>
     <id>LT.DigitalOffice.Kernel</id>
-    <version>1.1.26</version>
```

Comment:
I'm not sure that you can do this, have you ever released a nuget pack?

Reply:
Nope